base/utils/vmaas_test.go

```diff
@@ -0,0 +1,245 @@
+package utils
+
+import (
+	"app/base/vmaas"
+	"encoding/json"
+	"github.com/stretchr/testify/assert"
+	"reflect"
+	"testing"
+)
+
+func TestMergeUpdates(t *testing.T) {
+	var listA vmaas.UpdatesV2ResponseUpdateList
+	var listB vmaas.UpdatesV2ResponseUpdateList
+
+	err := json.Unmarshal(updatesBash, &listA)
+	assert.Nil(t, err)
+	err = json.Unmarshal(updatesBash2, &listB)
+	assert.Nil(t, err)
+
+	merged, err := mergeUpdates(listA, listB)
+	assert.Nil(t, err)
+
+	assert.Equal(t, 8, len(merged.GetAvailableUpdates()))
+	assert.Equal(t, true, updatesAscending(merged.GetAvailableUpdates()))
+}
+
+func TestMergeUpdatesWithDuplicates(t *testing.T) {
+	var listA vmaas.UpdatesV2ResponseUpdateList
+	var listB vmaas.UpdatesV2ResponseUpdateList
+
+	err := json.Unmarshal(updatesBash3, &listA)
+	assert.Nil(t, err)
+	err = json.Unmarshal(updatesBash4, &listB)
+	assert.Nil(t, err)
+
+	merged, err := mergeUpdates(listA, listB)
+	assert.Nil(t, err)
+
+	assert.Equal(t, 6, len(merged.GetAvailableUpdates()))
+	assert.Equal(t, true, updatesAscending(merged.GetAvailableUpdates()))
+}
+
+func TestMergeUpdatesWithALonger(t *testing.T) {
+	var listA vmaas.UpdatesV2ResponseUpdateList
+	var listB vmaas.UpdatesV2ResponseUpdateList
+
+	err := json.Unmarshal(updatesBash5, &listA)
+	assert.Nil(t, err)
+	err = json.Unmarshal(updatesBash6, &listB)
+	assert.Nil(t, err)
+
+	merged, err := mergeUpdates(listA, listB)
+	assert.Nil(t, err)
+
+	assert.Equal(t, 8, len(merged.GetAvailableUpdates()))
+	assert.Equal(t, true, updatesAscending(merged.GetAvailableUpdates()))
```

Comment:
If you see a code repetition it's probably a good candidate to put it into function.